src/YapepBase/BusinessObject/BoAbstract.php

```diff
@@ -61,7 +72,18 @@ public function __construct($keyPrefix = null) {
 	 * @return string
 	 */
 	protected function getKeyPrefix() {
-		return $this->keyPrefix . '.';
+		return $this->keyPrefix;
```

Comment:
This is a potentially breaking change

Reply:
It would be great to leave this method as is, and simply switch to the getKeyWithPrefix instead. We can deprecate this method if it's not used, but we shouldn't change it's behaviour without bumping the version. If we deprecate it now, we can remove it in 1.0.